src/compile/layout.ts

```diff
@@ -61,7 +61,7 @@ function getCellHeight(model: Model): LayoutValue {
       return model.config().cell.height;
     }
   }
-  return model.fieldDef(Y).scale.bandWidth;
+  return 21 /* config.scale.bandWidth */;
```

Comment:
They should refer to config.scale.width in the future.   Now it doesn't make sense to add config.scale unless we want to migrate all properties there.